packages/mwp-logger-plugin/package.json

```diff
@@ -24,7 +24,7 @@
   },
   "homepage": "https://github.com/meetup/meetup-web-platform#readme",
   "dependencies": {
-    "@google-cloud/logging-bunyan": "0.7.0",
+    "@google-cloud/logging-bunyan": "0.8.1",
```

Comment:
turns out this is a breaking change - I can update the code, but I'm curious what the motivation for the version bump was